.mailmap

```diff
@@ -248,6 +248,7 @@ Kostiantyn Wandalen <wandalen.me@gmail.com>
 Kris Kowal <kris.kowal@cixar.com>
 Kyle Robinson Young <kyle@dontkry.com>
 Lakshmi Swetha Gopireddy <lakshmiswethagopireddy@gmail.com> <lgopired@in.ibm.com>
+Lars-Magnus Skog <ralphtheninja@riseup.net> <lars.magnus.skog@gmail.com>
```

Comment:
I no longer use the gmail address.
```suggestion
Lars-Magnus Skog <ralphtheninja@riseup.net>
```

Reply:
In that case, the change here is what we want. The mailmap entry in this PR basically says "Don't use this gmail address. Use this riseup.net address instead if you see the gmail address in the commit log." In turn this removes the gmail address from the AUTHORS file when our tooling builds the AUTHORS file.

Reply:
Gotcha!